<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Check_Mainwindow</class>
 <widget class="QMainWindow" name="Check_Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>606</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,3">
    <item>
     <widget class="QWidget" name="leftWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1,1,1,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QPushButton" name="changebtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2196F3;  /* 蓝色背景 */
    border-radius: 10px;        /* 10px圆角 */
    color: white;               /* 白色文字 */
    padding: 6px;               /* 内边距 */
}

QPushButton:hover {
    background-color: #0b7dda;  /* 悬停时颜色变深 */
}

QPushButton:pressed {
    background-color: #005daa;  /* 点击时颜色更深 */
}</string>
         </property>
         <property name="text">
          <string>改数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deletebtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2196F3;  /* 蓝色背景 */
    border-radius: 10px;        /* 10px圆角 */
    color: white;               /* 白色文字 */
    padding: 6px;               /* 内边距 */
}

QPushButton:hover {
    background-color: #0b7dda;  /* 悬停时颜色变深 */
}

QPushButton:pressed {
    background-color: #005daa;  /* 点击时颜色更深 */
}</string>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearbtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2196F3;  /* 蓝色背景 */
    border-radius: 10px;        /* 10px圆角 */
    color: white;               /* 白色文字 */
    padding: 6px;               /* 内边距 */
}

QPushButton:hover {
    background-color: #0b7dda;  /* 悬停时颜色变深 */
}

QPushButton:pressed {
    background-color: #005daa;  /* 点击时颜色更深 */
}</string>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="paybtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2196F3;  /* 蓝色背景 */
    border-radius: 10px;        /* 10px圆角 */
    color: white;               /* 白色文字 */
    padding: 6px;               /* 内边距 */
}

QPushButton:hover {
    background-color: #0b7dda;  /* 悬停时颜色变深 */
}

QPushButton:pressed {
    background-color: #005daa;  /* 点击时颜色更深 */
}</string>
         </property>
         <property name="text">
          <string>支付</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="backbtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2196F3;  /* 蓝色背景 */
    border-radius: 10px;        /* 10px圆角 */
    color: white;               /* 白色文字 */
    padding: 6px;               /* 内边距 */
}

QPushButton:hover {
    background-color: #0b7dda;  /* 悬停时颜色变深 */
}

QPushButton:pressed {
    background-color: #005daa;  /* 点击时颜色更深 */
}</string>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addmemberbtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2196F3;  /* 蓝色背景 */
    border-radius: 10px;        /* 10px圆角 */
    color: white;               /* 白色文字 */
    padding: 6px;               /* 内边距 */
}

QPushButton:hover {
    background-color: #0b7dda;  /* 悬停时颜色变深 */
}

QPushButton:pressed {
    background-color: #005daa;  /* 点击时颜色更深 */
}</string>
         </property>
         <property name="text">
          <string>添加会员</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="CartgroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 购物车分组框样式 - 浅绿色系（与购物车表格风格呼应） */
QGroupBox#CartgroupBox {
    border: 1px solid #CCCCCC;
    border-radius: 4px;
    margin-top: 8px;
    background-color: #F8FBF8; /* 极浅的绿色背景，与购物车表格的浅绿风格协调 */
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="cartWidget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,1">
          <item>
           <widget class="QTableView" name="cartlistView">
            <property name="styleSheet">
             <string notr="true">/* 购物车表格整体样式 */
QTableView#cartlistView {
    background-color: #F5F9F4; 
    gridline-color: #DCE4E2; 
    selection-background-color: #C8E6C9; 
    selection-color: #000000; 
    outline: none; 
    /* 确保所有单元格都有背景色 */
    alternate-background-color: #F5F9F4;
}

/* 表头样式 */
QTableView#cartlistView QHeaderView::section {
    background-color: #E8F5E9; 
    color: #333333;
    font-weight: bold;
    border: 1px solid #DDDDDD; 
    padding: 4px;
    /* 确保表头第一列样式一致 */
    border-radius: 0;
}

/* 表格单元格样式 - 所有行和列 */
QTableView#cartlistView::item {
    background-color: #F5F9F4 !important; /* 强制所有单元格使用此背景色 */
    padding: 6px; 
    border: none; 
}

/* 选中行的单元格样式 */
QTableView#cartlistView::item:selected {
    background-color: #C8E6C9 !important; /* 强制选中行使用此背景色 */
    border: none; 
    outline: none; 
}

/* 特别处理第一列，确保其样式一致 */
QTableView#cartlistView::item:column(0) {
    background-color: #F5F9F4 !important; /* 确保第一列所有单元格使用此背景色 */
}</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>57</number>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="totalLabel">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>16</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#totalLabel {
    color: #1B5E20;  /* 主色调的深色，突出金额 */
    font-size: 20px;
    font-weight: bold;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="productgroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 商品区分组框样式 - 浅蓝色系（与商品列表风格呼应） */
QGroupBox#productgroupBox {
    border: 1px solid #CCCCCC;
    border-radius: 4px;
    margin-top: 8px;
    background-color: #F5F9FD; /* 极浅的蓝色背景，与商品列表的蓝色风格协调 */
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7,0">
       <item>
        <widget class="QWidget" name="categoryWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="categoryButtonsLayout"/>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="productlistWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">/* 商品浏览区整体样式（浅蓝底色） */
QListWidget#productlistWidget {
    background-color: #E3F2FD;  /* 最浅的蓝色背景（列表底色） */
    border: none;
    padding: 0;
    margin: 0px;
    outline: none;
}

/* 单个商品格样式（比列表深一度的蓝色） */
QListWidget#productlistWidget::item {
    background-color: #BBDEFB;  /* 商品底色（比列表深，比文字浅） */
    border-radius: 4px;
    padding: 15px 5px;
    min-width: 120px;
    min-height: 140px;
    text-align: center;
    font-family: &quot;微软雅黑&quot;;
    font-size: 16px;
    color: #1565C0;  /* 深蓝色文字（最深，确保清晰） */
    margin: 0px;
    border: 1px solid transparent;
}

/* 鼠标悬停效果（更深的蓝色） */
QListWidget#productlistWidget::item:hover {
    background-color: #90CAF9;  /* 悬停时比商品底色深 */
    border-color: #64B5F6;
}

/* 选中效果（最深的蓝色，突出选中状态） */
QListWidget#productlistWidget::item:selected {
    background-color: #64B5F6;  /* 选中时比悬停更深 */
    color: #0D47A1;  /* 选中时文字颜色加深，保持对比度 */
    border-color: #42A5F5;
    outline: none;
}

/* 禁用焦点指示器 */
QListWidget#productlistWidget::item:focus {
    outline: none;
    border-color: #42A5F5;
}</string>
         </property>
         <property name="textElideMode">
          <enum>Qt::TextElideMode::ElideNone</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::ResizeMode::Adjust</enum>
         </property>
         <property name="gridSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="viewMode">
          <enum>QListView::ViewMode::IconMode</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="historyBtn">
           <property name="toolTip">
            <string>历史记录</string>
           </property>
           <property name="icon">
            <iconset resource="../../Icon.qrc">
             <normaloff>:/LogIn/icon/LogIn/history.png</normaloff>:/LogIn/icon/LogIn/history.png</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>productgroupBox</zorder>
   <zorder>CartgroupBox</zorder>
   <zorder>leftWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Icon.qrc"/>
 </resources>
 <connections/>
</ui>
